### ISSUE TITLE
After training, only the class of my dataset was trained properly.

### ISSUE THREAD

**Issue by sankim90** [pjreddie/darknet#1466]
Hi

I want to create a model with 9 classes.

base model: darknet53.conv.74

The Training dataset uses voc, coco, myown data.

train.data is
 classes= 9
 train = data/mydata/finetune_train.txt
 valid = data/mydata/finetune_valid.txt
 names = data/NDM.names
 backup = NDM_finetune/

finetune_train.txt is
.
.
/home/sankim/darknet/data/mydata/images/train2018/mydata_train2018_001100.jpg
/home/sankim/darknet/data/coco/images/train2014/COCO_train2014_000000000009.jpg
/home/sankim/darknet/data/VOC/VOCdevkit/VOC2012/JPEGImages/2011_003244.jpg
.
.
.

directory is
data/mydata/images
data/mydata/labels
data/coco/images
data/coco/labels
data/voc/images
data/voc/labels

my dataset contains only dogs, traffic lights and traffic signals.(3 classes)
The coco, voc dataset contains the remaining 6 classes.

After training, only the class of my dataset was trained properly.

Why??

I expect the following.
1. voc, coco, and mydata must all be in one directory.
2. There is too much data in the mydata set or the image is too large.

Please help.

**Comment by SalmonMode** [comment]
I'm pretty new to this myself, but I believe you can only point it to one file for the training data and valid references respectively, and you pointed it at `data/mydata/finetune_train.txt` and `data/mydata/finetune_valid.txt`. From what I understand though, if you want to only train on 9 things, you can go through the normal training approach, using the normal data, but just alter some config files before training. I found some general instructions here #142, and I'm currently training on 7 classes (no idea if it will work though).
